blockchain/transact.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+# Get the input private key 
+read -s -p "Enter Private Key: " private_key
+
+# Import the key into the EOSIO wallet
+cleos wallet import --private-key $private_key
+
+echo $private_key
+
+# Get the wallet alias
+read -s -p "Enter alias url: " alias_url
```

Comment:
"Enter blockchain API url (e.g. https://blockchain-api.pangea.web4.world)"

- this is clearer